azurelinuxagent/common/event.py

```diff
@@ -383,6 +383,10 @@ def __init__(self):
         self._common_parameters.append(TelemetryEventParam(CommonTelemetryEventSchema.VMId, "VMId_UNINITIALIZED"))
         self._common_parameters.append(TelemetryEventParam(CommonTelemetryEventSchema.ImageOrigin, 0))
 
+    @staticmethod
+    def _get_vm_arch():
+        return platform.machine()
```

Comment:
can this ever raise an exception? should we catch and return 'unknown' on failure?

Reply:
Updated to catch and return 'unknown' on failure